src/Core/Tool/Filesystem.php

```diff
@@ -15,7 +15,7 @@
 
 interface Filesystem
 {
-	public function putStream($path, $resource, $visibility = null);
+	public function putStream($path, $resource, array $config = []);
```

Comment:
The version of flysystem changed to support the new adapters, these was the only change between version. In flysystem - at least that effects us